### DISCUSSION TITLE
JDemetra+ Universe Blog - Webinar on R tools for JDemetra+ from v2 to v3

### DISCUSSION THREAD

**Discussion by giscus** [annasmyk/JD_Quarto_Blog#2] [General]
# JDemetra+ Universe Blog - Webinar on R tools for JDemetra+ from v2 to v3

https://jdemetra-universe-blog.netlify.app/posts/webinar_rjd_dec22/

**Comment by annasmyk** [comment]
I am looking forward to discovering the 23 R packages of version 3.
